transports/http/endpoints/api/profile/endpoints.go

```diff
@@ -0,0 +1,31 @@
+package profile
+
+import (
+	"net/http/pprof"
+
+	"github.com/bitcoin-sv/pulse/service"
+	router "github.com/bitcoin-sv/pulse/transports/http/endpoints/routes"
+	"github.com/gin-gonic/gin"
+)
+
+// RegisterPprofEndpoints registers routes that are part of pprof.
+func NewHandler(s *service.Services, pprofEnabled bool) router.PprofEndpoints {
```

Comment:
IMO the handler should be added conditionally

Reply:
Fixed